### ISSUE TITLE
Artefacts of generation

### ISSUE THREAD

**Issue by mario-minati** [reneeb/MySQL-Workbench-DBIC#6]
Hello Renée,

in many of our generated result classes there are the following artefacts:
`$VAR1 = {};`

Actually we cannot see any logic when this happens, example:

```
package Glue::DB::Schema::Core::Result::StorageDocument;

# ABSTRACT: Result class for storage_document

use strict;
use warnings;
use base qw(DBIx::Class);

our $VERSION = 5;

__PACKAGE__->load_components( qw/PK::Auto Core/ );
__PACKAGE__->table( 'storage_document' );
__PACKAGE__->add_columns(
    id => {
        data_type          => 'INT',
        is_auto_increment  => 1,
        is_numeric         => 1,
        retrieve_on_insert => 1,
    },
    project_id => {
        data_type          => 'INT',
        is_nullable        => 1,
        is_numeric         => 1,
        is_foreign_key     => 1,
              $VAR1 = {};

    },
    description => {
        data_type          => 'MEDIUMTEXT',
        is_nullable        => 1,
    },
    keytext => {
        data_type          => 'TEXT',
    },
    process_module => {
        data_type          => 'VARCHAR',
        is_nullable        => 1,
        size               => 512,
              $VAR1 = {};

    },
    storage_document_type_id => {
        data_type          => 'INT',
        is_numeric         => 1,
        is_foreign_key     => 1,
              $VAR1 = {};

    },
    access_role_id => {
        data_type          => 'INT',
        is_numeric         => 1,
        is_foreign_key     => 1,
              $VAR1 = {};

    },
    locked => {
        data_type          => 'TINYINT',
        size               => 1,
              $VAR1 = {};

    },

);
__PACKAGE__->set_primary_key( qw/ id / );


__PACKAGE__->has_many(storage_file => 'Glue::DB::Schema::Core::Result::StorageFile',
             { 'foreign.storage_document_id' => 'self.id' });


__PACKAGE__->belongs_to(access_role => 'Glue::DB::Schema::Core::Result::AccessRole',
             { 'foreign.id' => 'self.access_role_id' });

__PACKAGE__->belongs_to(project => 'Glue::DB::Schema::Core::Result::Project',
             { 'foreign.id' => 'self.project_id' });

__PACKAGE__->belongs_to(storage_document_type => 'Glue::DB::Schema::Core::Result::StorageDocumentType',
             { 'foreign.id' => 'self.storage_document_type_id' });




# ---
# Put your own code below this comment
# ---

# ---

1;
```

Greets,

Mario

**Comment by reneeb** [comment]
Fixed in version 1.11

**Comment by mario-minati** [comment]
Thanks for the quick fix :-) 👍
